package.json

```diff
@@ -38,8 +38,7 @@
     "rollup-plugin-babel": "^4.4.0"
   },
   "dependencies": {
-    "@cesarbr/knot-cloud-sdk-js-amqp": "^1.0.0",
-    "lodash": "^4.17.13"
```

Comment:
Since you removed `lodash` here, yarn.lock should be updated?

Reply:
Good question! I'll make a rebase and update `yarn.lock` from commits based on `package.json` changes

Reply:
Done!